src/test/groovy/edu/ie3/datamodel/io/factory/input/participant/EvcsInputFactoryTest.groovy

```diff
@@ -44,9 +43,9 @@ class EvcsInputFactoryTest extends Specification  implements FactoryTestHelper {
 			"operatesuntil"   : "2019-12-31T23:59:00+01:00[Europe/Berlin]",
 			"id"              : "TestID",
 			"qcharacteristics": "cosPhiFixed:{(0.0,1.0)}",
-			"type"          : "Household",
-			"chargingpoints"         : "4",
-			"cosphirated"          : "0.95",
+			"type"            : "Household",
```

Comment:
Is there a place, where the parsing of _all_ permissible input it tested?

Reply:
Yes, that's what happens in [ChargingPointTypeUtilsTest](https://github.com/ie3-institute/PowerSystemDataModel/blob/dev/src/test/groovy/edu/ie3/datamodel/models/input/system/type/chargingpoint/ChargingPointTypeUtilsTest.groovy)